manifests/w/WorkFlowy/WorkFlowy/4.0.2302081114.991/WorkFlowy.WorkFlowy.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: WorkFlowy.WorkFlowy
+PackageVersion: 4.0.2302081114.991
```

```suggestion
PackageVersion: 4.0.2302081114
```
Must match the version shown in Control Panel / Apps&Features